packages/image_picker/image_picker_platform_interface/lib/src/types/lost_data_response.dart

```diff
@@ -34,7 +34,7 @@ class LostDataResponse {
   /// The file that was lost in a previous [pickImage] or [pickVideo] call due to MainActivity being destroyed.
   ///
   /// Can be null if [exception] exists.
-  final File file;
+  final File? file;
```

Comment:
I think this whole file can also be removed. Isn't it only used as a return value from deprecated methods? (It'll need to be removed from wherever it's `export`ed as well, and added to the breaking changes docs.)

Reply:
done